pkg/kubelet/network/hostport/hostport_syncer_test.go

```diff
@@ -215,6 +215,189 @@ func TestOpenPodHostports(t *testing.T) {
 	}
 }
 
+func TestOpenIPv6PodHostports(t *testing.T) {
+	fakeIPTables := NewFakeIPTables(ProtocolIpv6)
+	fakeOpener := NewFakeSocketManager()
+
+	h := &hostportSyncer{
+		hostPortMap: make(map[hostport]closeable),
+		iptables:    fakeIPTables,
+		portOpener:  fakeOpener.openFakeSocket,
+	}
+
+	tests := []struct {
+		mapping *PodPortMapping
```

Comment:
Same comment here, about a test case name to indicate which one failed, and use it in the Fatalf() calls below.